src/dags/common/db.py

```diff
@@ -18,3 +19,18 @@ def get_engine(postgres_conn_id="postgres_default") -> Engine:
         return create_engine(f"postgresql://{user}:{password}@{host}:{port}/{db}")
     except SQLAlchemyError as e:
         raise AirflowException(str(e)) from e
+
+
+def get_ora_engine(oracle_conn_id="oracle_default") -> Engine:
+    connection = OracleHook().get_connection(oracle_conn_id)
+    user = connection.login
+    password = connection.password
+    host = connection.host
+    port = connection.port
+    db = connection.schema
+
+    try:
+        uri = f"oracle+cx_oracle://{user}:{password}@{host}:{port}/{db}?encoding=UTF-8&nencoding=UTF-8"
```

Comment:
Wellicht is hier DbApiHook.get_uri() nog voor te gebruiken? Of werkt dat wellicht niet met Oracle?